other/bootstrap_daemon/tox-bootstrapd.service

```diff
@@ -4,11 +4,8 @@ After=network.target
 
 [Service]
 Type=forking
-PermissionsStartOnly=true
-ExecStartPre=-/bin/mkdir /var/run/tox-bootstrapd -p
-ExecStartPre=/bin/chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd
 WorkingDirectory=/var/lib/tox-bootstrapd
-ExecStart=/usr/local/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
+ExecStart=/usr/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
```

Comment:
How will I get this reverted? I just make another commit with the path fixed?

Anyway, the systemd files get copied manually. Maybe I better make some amendments in the readme at other/bootstrap_daemon/README.md

What do you think?